src/test/java/nextstep/subway/line/domain/LineTest.java

```diff
@@ -124,44 +126,63 @@ void addSection_WhenBothStationAlreadyExists_ThenFail() {
     @DisplayName("상행역과 하행역 둘 중 하나도 포함되어있지 않으면 추가할 수 없음")
     @Test
     void addSection_WhenBothStationDoesNotExists_ThenFail() {
-        //given
-        //강남----------정자
+        // given
+        // 강남----------정자
 
-        //when
-        //양재------------청계산입구 추가
-        //then
-        //BothStationNotExistsException
+        // when
+        // 양재------------청계산입구 추가
+        // then
+        // BothStationNotExistsException
         assertThatThrownBy(() -> {
             신분당선.addSection(new Section(신분당선, 양재역, 청계산입구역,12));
         }).isInstanceOf(BothStationNotExistsException.class);
     }
 
+    @DisplayName("하행 종점역 삭제")
     @Test
     void removeSection() {
-        신분당선.addSection(new Section(신분당선, 정자역, 미금역, 7));
-
-        int before = 신분당선.getStations().size();
-        신분당선.removeSection(미금역.getId());
+        // given
+        //강남----------정자-----미금
+        신분당선.addSection(new Section(신분당선, 정자역, 미금역, 5));
 
-        int after = 신분당선.getStations().size();
+        // when
+        //정자역 삭제
+        신분당선.removeSection(3L);
```

Comment:
```suggestion
        신분당선.removeSection(미금역.getId());
```
이러면 가독성이 더 좋아질 것 같습니다 :)